prusti-viper/src/verifier.rs

```diff
@@ -422,8 +422,10 @@ fn verify_programs(env: &Environment, programs: Vec<vir::Program>)
         stopwatch.start_next("attach current thread to the JVM");
         let viper_thread = viper.attach_current_thread();
         stopwatch.finish();
+        let cache_loc = config::cache_dir() + "data.json";
```

Comment:
`PathBuf::from(...).join(...)` is more robust.

Also, what about renaming `cache_dir` to `cache_path`, making the default `"cache/data.json"`? That way, the user (or the IDE) can use e.g. `/tmp/prusti_cache.json`.